index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Portfolio</title>
+	<link rel="stylesheet" type="text/css" href="style.css">
+	<link href='http://fonts.googleapis.com/css?family=Abel' rel='stylesheet' type='text/css'>
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.3.0/css/font-awesome.min.css">
+	<link href='http://fonts.googleapis.com/css?family=Rajdhani:300,400,500,600,700&subset=latin,latin-ext' rel='stylesheet' type='text/css'>
+	<link href='http://fonts.googleapis.com/css?family=Fira+Sans:300,400,500,700' rel='stylesheet' type='text/css'>
+	<link href='http://fonts.googleapis.com/css?family=Raleway:400,200,600,100,700' rel='stylesheet' type='text/css'>
+</head>
+<body>
+	<header>
+		<nav id="nav-container"></nav>
+		<nav id="nav-elements">
+			<ul>
+				<li id="hamburger-icon"><i class="fa fa-bars"></i></li>
+				<li><a href="#">Contact</a></li>
+				<li><a href="#">About</a></li>
+				<li><a href="#">Home</a></li>
+				
+			</ul>
+		</nav>
+	</header>
+	<section id="intro" class="intro-home">
+		<div id="intro-contain">
+			<img src="images/logo_header-01.svg">
+			<h2><i>Vision & Execution</i></h2>
+		</div>
+		<div id="bgvideo">
+			<div class="videoBG_wrapper">
+				<div id="bgvideo"></div>
+				
+				<div style="position: absolute; z-index: 0; top: 0px; left: 0px; width:100%; opacity:1;">
+					<video poster="images/ballons.jpg" autoplay="autoplay" src="video/balloons.webm" style="position: absolue; z-index:0; top:0px; left: 0px; min-width:100%; min-height:100%;" loop="true"></video>
```

Comment:
great job with the video! I definitely would like you to show the class how you did this and lead a mini-tutorial on doing background videos.